### PR TITLE
Commands and Clear

### PR DESCRIPTION
**Pull request by Kwayzu**
added clear button without dom manipulation as per ben.  Added commands !duck and !roll as we all as an alias for lmgtfy as !google

### CODE REVIEW

File: webbage.chat.web/app/components/rooms/views/room.tpl.html [hunk 1]
```diff
@@ -43,12 +43,24 @@
...
+               ng-click="playSound=!playSound"
+               ng-class="{ 'fa-volume-up': playSound, 'fa-volume-off': !playSound}"></i>
+        </button>
+
+        <!--<i class="fa fa-2x"
            ng-class="{ 'fa-volume-up': playSound, 'fa-volume-off': !playSound}"
-           ng-click="playSound=!playSound"></i>
+           ng-click="playSound=!playSound" style="padding-left:40px"></i>-->
     </div>
 </div>
```
Comment (wolvenhaven) [inline webbage.chat.web/app/components/rooms/views/room.tpl.html]: missing empty line EOF

File: webbage.chat.web/app/components/rooms/controllers/roomCtrl.js [hunk 2]
```diff
@@ -196,6 +196,10 @@
 
                     sentMessages.push(message);
                     sentMessagesIndex = sentMessages.length;
+                    if ($scope.isMessageCode)
+                    {
+                        $scope.isMessageCode = false;
```
Comment (bamblack) [inline webbage.chat.web/app/components/rooms/controllers/roomCtrl.js]: I like this, good catch.

File: webbage.chat.web/webbage.chat.web.csproj [hunk 3]
```diff
@@ -14,7 +14,7 @@
     <RootNamespace>webbage.chat.web</RootNamespace>
     <AssemblyName>webbage.chat.web</AssemblyName>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
-    <UseIISExpress>false</UseIISExpress>
+    <UseIISExpress>true</UseIISExpress>
```
Comment (wolvenhaven) [inline webbage.chat.web/webbage.chat.web.csproj]: Why is this changing?

File: webbage.chat.web/app/components/rooms/views/room.tpl.html [hunk 4]
```diff
@@ -43,12 +43,24 @@
...
                 ng-click="isMessageCode = !isMessageCode">
             code
         </button>
-        <i class="fa fa-2x"
+        <button type="button"
+                class="btn clouds-font wet-asphalt-background"
+                ng-click="clearChat()">
+            clear
+        </button>
+        <button type="button"
```
Comment (bamblack) [inline webbage.chat.web/app/components/rooms/views/room.tpl.html]: Not sure I like making this a button. We need to figure out how to position it properly with added buttons.

Perhaps icons like this should be moved above the chat buttons. But it shouldn't be made into a button.

**Comment by bamblack** [comment]
In light of the `.csproj` change and discussing it with Kway, I'm doing to reject this PR. He's going to change over to using full IIS on his machine and once the changes are reverted we can make another PR. Other than that this looks like a good commit. We can discuss how we want to arrange the buttons [here](https://github.com/webbage/webbage.chat/issues/73).

**Pull request closed without merging**